lms/djangoapps/certificates/api.py

```diff
@@ -448,25 +448,18 @@ def get_certificate_url(user_id=None, course_id=None, uuid=None):
     if not course:
         return url
 
-    if has_html_certificates_enabled(course) and has_any_active_web_certificate(course):
+    if has_html_certificates_enabled(course):
         url = _certificate_html_url(user_id, course_id, uuid)
     else:
         url = _certificate_download_url(user_id, course_id)
     return url
 
 
-def has_any_active_web_certificate(course):
-    if hasattr(course, 'has_any_active_web_certificate'):
-        return course.has_any_active_web_certificate
-
-    return get_active_web_certificate(course)
-
-
 def get_active_web_certificate(course, is_preview_mode=None):
     """
     Retrieves the active web certificate configuration for the specified course
     """
-    certificates = getattr(course, 'certificates', {})
+    certificates = getattr(course, 'certificates', '{}')
```

Comment:
Why we need to pass it as string?

Reply:
@ahsan-ul-haq It's a revert PR and was implemented like this, I figured it out while some tests started failing due to my changes then I fixed it.